x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

```diff
@@ -4,9 +4,11 @@
 <titleabbrev>Set up basic security</titleabbrev>
 ++++
 
-After adding password protection in the <<security-minimal-setup,minimal security configuration>>, you'll need to configure Transport Layer Security
-(TLS). The transport layer handles all internal communication between nodes in
-your cluster.
+When you start {es} for the first time, passwords are generated for the `elastic`
+user and Transport Layer Security (TLS) is automatically configured for you. If 
```

Comment:
```suggestion
user and TLS is automatically configured for you. If 
```

I think that most people are familiar with the abbreviation and those who are not won't get any insights from the words it stands for

Reply:
++ I'll make that change.